src/main/java/uk/gov/hmcts/reform/juddata/scheduler/JrdUserProfileDataLoadScheduler.java

```diff
@@ -42,9 +48,16 @@ public void postConstruct() throws Exception {
 
     @Scheduled(cron = "${scheduler.camel-route-config}")
     public void runJrdScheduler() {
-        camelContext.getGlobalOptions().remove(IS_EXCEPTION_HANDLED);
-        camelContext.getGlobalOptions().remove(SCHEDULER_STATUS);
-        dataLoadUtil.setGlobalConstant(camelContext, JUDICIAL_USER_PROFILE_ORCHESTRATION);
-        producerTemplate.sendBody(startRoute, "starting JRD orchestration");
+        try {
+            camelContext.getGlobalOptions().remove(IS_EXCEPTION_HANDLED);
+            camelContext.getGlobalOptions().remove(SCHEDULER_STATUS);
+            dataLoadUtil.setGlobalConstant(camelContext, JUDICIAL_USER_PROFILE_ORCHESTRATION);
+            producerTemplate.sendBody(startRoute, "starting JRD orchestration");
+        } catch (Exception ex) {
+            log.error("::judicial-user-profile-orchestration route failed::",  ex.getMessage());
```

Comment:
could you please print the whole exception stack-trace here instead of message?

Reply:
https://stackoverflow.com/questions/10477607/avoid-printstacktrace-use-a-logger-call-instead

Reply:
I mean print ex object instead of ex.message()